<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	border-radius: 10px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>651</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsVidget</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="out">
           <property name="styleSheet">
            <string notr="true">QLabel{
border: 2px solid gray;
	border-radius: 10px;
font-size: 14px;
	font:	Roboto-Light;
}</string>
           </property>
           <property name="text">
            <string>Путь: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="out_2">
           <property name="styleSheet">
            <string notr="true">QLabel{
border: 2px solid gray;
	border-radius: 10px;
font-size: 14px;
	font:	Roboto-Light;
}</string>
           </property>
           <property name="text">
            <string>Длина пути:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 10px;
    color: white;
    transition: .2s linear;
    background: #0B63F6;
	font-size: 14px;
	font:	Roboto-Light;
}
QPushButton:pressed{
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Создать матрицу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 10px;
    color: white;
    transition: .2s linear;
    background: #0B63F6;
	font-size: 14px;
	font:	Roboto-Light;
}
QPushButton:pressed{
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Отправить данные</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GraphButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 10px;
    color: white;
    transition: .2s linear;
    background: #0B63F6;
	font-size: 14px;
	font:	Roboto-Light;
}
QPushButton:pressed{
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Создать Граф</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
		border: 2px solid gray;
		border-radius: 10px;
		padding: 0 8px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 2px solid gray;
	border-radius: 10px;
font-size: 14px;
	font:	Roboto-Light;
}</string>
    </property>
    <property name="text">
     <string>Размер матрицы</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 10px;
    color: white;
    transition: .2s linear;
    background: #0B63F6;
	font-size: 14px;
	font:	Roboto-Light;
}
QPushButton:pressed{
	border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Кратчайший путь</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
